src/statsmodel.jl

```diff
@@ -106,6 +106,71 @@ function formula end
 formula(m::TableStatisticalModel) = m.mf.f
 formula(m::TableRegressionModel) = m.mf.f
 
+"""
+    termnames(model::StatisticalModel)
+
+Return the names of terms used in the formula of `model`.
+
+This is a convenience method for `termnames(formula(model))`.
+
+For `RegressionModel`s with only continuous predictors, this is the same as
+`(responsename(model), coefnames(model))`.
+
+For models with categorical predictors, the returned names reflect
+the variable name and not the coefficients resulting from
+the choice of contrast coding.
+"""
+termnames(model::StatisticalModel) = termnames(formula(model))
+"""
+    termnames(t::FormulaTerm)
+
+Return a two-tuple of `termnames` applied to the left and
+right hand sides of the formula.
```

Comment:
Given that we guarantee that this returns a two-tuple and that the method above is a shorthand for `termnames(formula(model))`, shouldn't we also say in the docstring for that method that it returns a two-tuple? (I hadn't realized, but at least currently formulas are required to have both a LHS and a RHS.)